src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -183,19 +183,24 @@ DlgPrefInterface::DlgPrefInterface(
 #ifdef MIXXX_USE_QML
     if (CmdlineArgs::Instance().isQml()) {
         mulitSamplingComboBox->clear();
-        mulitSamplingComboBox->addItem(tr("Disabled"), 0);
-        mulitSamplingComboBox->addItem(tr("2x MSAA"), 2);
-        mulitSamplingComboBox->addItem(tr("4x MSAA"), 4);
-        mulitSamplingComboBox->addItem(tr("8x MSAA"), 8);
-        mulitSamplingComboBox->addItem(tr("16x MSAA"), 16);
-
-        m_multiSampling = m_pConfig->getValue(ConfigKey(kPreferencesGroup, kMultiSamplingKey), 4);
-        int mulitSamplingIndex = mulitSamplingComboBox->findData(m_multiSampling);
+        mulitSamplingComboBox->addItem(tr("Disabled"), mixxx::MultiSamplingMode::DISABLED);
+        mulitSamplingComboBox->addItem(tr("2x MSAA"), mixxx::MultiSamplingMode::TWO);
+        mulitSamplingComboBox->addItem(tr("4x MSAA"), mixxx::MultiSamplingMode::FOUR);
+        mulitSamplingComboBox->addItem(tr("8x MSAA"), mixxx::MultiSamplingMode::EIGHT);
+        mulitSamplingComboBox->addItem(tr("16x MSAA"), mixxx::MultiSamplingMode::SIXTEEN);
+
+        m_multiSampling = m_pConfig->getValue(
+                ConfigKey(kPreferencesGroup, kMultiSamplingKey),
+                static_cast<int>(mixxx::MultiSamplingMode::FOUR));
+        int mulitSamplingIndex = mulitSamplingComboBox->findData(
+                static_cast<mixxx::MultiSamplingMode::>(m_multiSampling));
```

Comment:
please use my enum overload ;)
```suggestion
        m_multiSampling = m_pConfig->getValue<mixxx::MultiSamplingMode>(
                ConfigKey(kPreferencesGroup, kMultiSamplingKey),
                mixxx::MultiSamplingMode::FOUR);
        int mulitSamplingIndex = mulitSamplingComboBox->findData(m_multiSampling);
```

Reply:
> please use my enum overload ;)

Ah, yes, I remembered it but didn't find the PR/commit so I resorted to the casting madness